clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -854,21 +854,19 @@ public void stopConsumptionAndWait(
         LOGGER.error(
             "Partition: {} of store: {} is still consuming after waiting for it to stop for {} seconds.",
             partitionId,
-            veniceStore.getStoreVersionName(),
+            topicName,
             numRetries * sleepSeconds);
       } catch (InterruptedException e) {
         LOGGER.warn("Waiting for partition to stop consumption was interrupted", e);
         currentThread().interrupt();
       }
     } else {
-      LOGGER.warn(
-          "Partition: {} of topic: {} is not consuming, skipped the stop consumption.",
-          partitionId,
-          veniceStore.getStoreVersionName());
+      LOGGER.warn("Partition: {} of topic: {} is not consuming, skipped the stop consumption.", partitionId, topicName);
     }
     resetConsumptionOffset(veniceStore, partitionId);
-    if (!ingestionTaskHasAnySubscription(veniceStore.getStoreVersionName())) {
-      shutdownStoreIngestionTask(veniceStore);
+    if (!(isIsolatedIngestion || ingestionTaskHasAnySubscription(topicName))) {
```

Comment:
Can you leave some logging when `II` is on?
Something like `the StoreIngestionTask instance will be kept for the access from main process` indicates a different behavior.